ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +244,35 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Physical Volume IO Stats:
+                        units: per sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            RRQM:
+                                dpName: diskstats_numReadsMerged
+                                lineType: AREA
+                                rpn: "${here/sysUpTime},/,100,*"
+                            WRQM:
+                                dpName: diskstats_numWritesMerged
+                                lineType: AREA
+                                rpn: "${here/sysUpTime},/,100,*"
+                            Reads:
+                                dpName: diskstats_totalReads
```

Comment:
Reads/Writes merged, Reads/Writes, and Sectors Read/Written probably belong on three different graphs since the units are different for each.